release.nix

```diff
@@ -15,19 +15,22 @@ let
       PKG_IGNORE_TAG = 1;
       buildPhase = ''
         cp ${./package.json} package.json
+        mkdir -p output/prebuilds
+        cp -r node_modules/leveldown/prebuilds/linux-${arch} output/prebuilds/linux-${arch}
         pkg . \
           --targets linux-${arch} \
           --no-bytecode \
-          --output out
+          --public-packages "*" \
+          --output output/TypeScript-Demo-Lib
       '';
       installPhase = ''
-        cp out $out
+        cp -r output $out
       '';
       dontFixup = true;
     };
   buildExe = arch:
     stdenv.mkDerivation rec {
-      name = "${utils.basename}-${version}-win32-${arch}.exe";
+      name = "${utils.basename}-${version}-win32-${arch}";
```

Comment:
Why change this? The name of the windows executable has an extension of `.exe`.

Reply:
The output is a directory now that includes the .exe

Reply:
You should leave it as the actual executable. Is there other files necessary for the actual `$out`?

This is not a nix package, so the executable as `$out` is totally fine.

Plus this actually is integrated into the Gitlab CI/CD which expects a single file as `$out`. Changing it to a directory adds extra work to do there.

Reply:
We need to have the output as a directory since we need to include the prebuilds for leveldown.
we can have it as a zip file if needed.

Reply:
Can this be resolved now back to the way it was?